Framework/DataHandling/src/LoadNexusProcessed.cpp

```diff
@@ -1721,8 +1721,13 @@ void LoadNexusProcessed::loadNonSpectraAxis(
     }
   } else if (axis->isText()) {
     NXChar axisData = data.openNXChar("axis2");
-    axisData.load();
-    std::string axisLabels(axisData(), axisData.dim0());
+    std::string axisLabels;
+    try {
+      axisData.load();
+      axisLabels = std::string(axisData(), axisData.dim0());
+    } catch (std::exception &) {
+      axisLabels = "";
```

Comment:
I'm thinking do we want to log a `warning` or `information`, I think the best place for the warning would be during saving?